### PR TITLE
feat(feed): Added feed cog

### PR DESCRIPTION
**Pull request by issy**
# Initial Checklist
- [x] Has @tigattack and/or @kdavis been added as a reviewer?
- [ ] If applicable, have the relevant project(s), milestone(s), and label(s) been applied?
- [x] If applicable, have you added details of the cog to the readme as per [README.md](https://github.com/rHomelab/LabBot-Cogs/blob/develop/README.md#cog-summaries)?

# Details
* **Does this resolve an issue?**
N/A

### New Features
* Adds [p]feed command, which allows users to feed each other

### CODE REVIEW

File: feed/feed.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        '🥃',
+        '🥤',
+        '🧃',
+        '🧉',
+        '🧊']
+
+class FeedCog(commands.Cog):
+    """Feed Cog"""
+
+    def __init__(self, bot):
```
Comment (kdavis) [inline feed/feed.py:121]: Is this constructor redundant?

File: README.md [hunk 2]
```diff
@@ -65,6 +65,10 @@ This cog will allow members to send a report into a channel where it can be revi
 
 This cog will allow users to prove they're not a bot by having to read rules and complete an action. They will then be given the verified role if they can complete this.
 
+### Feed
+
+This cog allows users to feed each other
```
Comment (kdavis) [inline README.md:70]: This is not in alphabetical order

Comment (kdavis) [reply README.md:70]: Also missing the ToC addition and Cog Summaries link

**Review by kdavis — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (issy) [reply README.md:70]: > ToC addition and Cog Summaries link
What's this? Please explain?

Comment (kdavis) [reply README.md]: In README, there's two more sections, "Table of Contents" and "Cog Summaries" - Feed needs to be added to both of these lists too 😄

Comment (issy) [reply README.md]: Ah I see it now. Added to both

File: feed/feed.py (hunk 1 above)
Comment (issy) [reply feed/feed.py:121]: Let's find out!

**Review by kdavis — APPROVED** [review]

**Pull request merged**